prover/zkevm/prover/ecdsa/adress.go

```diff
@@ -17,6 +17,17 @@ import (
 	"github.com/consensys/linea-monorepo/prover/zkevm/prover/hash/generic"
 )
 
+const (
+	// addressHiBytes is the size of the leftover from trimmed addressHi part (in bytes).
+	addressHiBytes = 4
```

Comment:
If not mistaken, every limb of an address will have to store that number of bytes

Reply:
Ah I think, I understand. You basically applied a direct strategy where you just split everything there is in place over 16 bits. Before there used to be a difference because the LO limb was 128 bits and the high one was 32. And now, they are all broken down in 16 bits sub-limbs.

But still, there is no need to differentiate between the high and the low part at this point IMO. Would be simpler to say that an address is 10 limbs of 16 bits

Reply:
Oh ok, wrong again. It's the trimmed vs untrimmed part. Ok that makes sense